### ISSUE TITLE
output description not being generated

### ISSUE THREAD

**Issue by edgar-humberto** [segmentio/terraform-docs#29]
The description for outputs are not being generated on version 0.1.0

    output "asg-blue-name" {
      description = "Name of Blue autoscaling group."
      value = "${aws_autoscaling_group.app-asg-blue.name}"
    }
   

    | Name | Description |
    |------|-------------|
    | asg-blue-name |  |
    | asg-green-name |  |

**Comment by ericdahl** [comment]
duplicate of #24 and should be fixed with #25. It would be nice to get a release which includes this

**Comment by metmajer** [comment]
@edgar-humberto @ericdahl I have verified that `terraform-docs` now generates correct output for your example:

```
output "asg-blue-name" {
  description = "Name of Blue autoscaling group."
  value = "${aws_autoscaling_group.app-asg-blue.name}"
}
```

```
## Outputs

| Name | Description |
|------|-------------|
| asg-blue-name | Name of Blue autoscaling group. |
```
